src/app/games/encapsule/encapsule.component.ts

```diff
@@ -171,20 +180,17 @@ export class EncapsuleComponent extends RectangularGameComponent<EncapsuleRules,
     }
 
     public getPieceRadius(piece: EncapsulePiece): number {
-        switch (piece.getSize()) {
-            case Size.BIG:
-                return (this.SPACE_SIZE / 2) - (1 * this.STROKE_WIDTH);
-            case Size.MEDIUM:
-                return (this.SPACE_SIZE / 2) - (2 * this.STROKE_WIDTH) - 3;
-            default:
-                Utils.expectToBe(piece.getSize(), Size.SMALL);
-                return (this.SPACE_SIZE / 2) - (3 * this.STROKE_WIDTH) - 6;
-        }
+        const size: number = piece.getSize();
+        // The aim is that the inter-piece space is half a piece stroke
+        // And the the radius is used like this (from center to outside):
+        // empty - stroke - stroke - empty - stroke - stroke
+        // And stick to the square's inner-edge
```

Comment:
"and the the"
puis, tu dis "le rayon doit être utilisé comme ceci : <aucune mention du rayon>"
et la dernière ligne est ambigue, on parle du rayon ou du inter-piece space ?

Reply:
aucun des deux, square's inner-edge c'est pour dire que le cercle le plus grand sera inscrit au rectangle

Reply:
ouais, pas clair pantoute, j'te propose de rephraser !

Reply:
Commente ça au vrai endroit, tu parles sur le passé tout est déjà réglé !